packages/fftw3/fftw3.0.7.2/opam

```diff
@@ -15,7 +15,19 @@ build: [
       "--localstatedir=%{lib}%/fftw3/var"
       "--libdir=%{lib}%/fftw3/lib"
       "--includedir=%{lib}%/fftw3/include"
-      "--datarootdir=%{lib}%/fftw3/share"]
+      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
+  ["./configure" 
+      "CFLAGS=-I/usr/local/include"
+      "LDFLAGS=-L/usr/local/lib"
```

Comment:
Yeah. I was having a weird path issue with OSX's `clang` but I fixed it
locally in fact. Sorry for the noise..

2016-01-28 11:00 GMT-06:00 Christophe Troestler <notifications@github.com>:

> In packages/fftw3/fftw3.0.7.2/opam
> <https://github.com/ocaml/opam-repository/pull/5519#discussion_r51151104>:
>
> > @@ -15,7 +15,19 @@ build: [
> >        "--localstatedir=%{lib}%/fftw3/var"
> >        "--libdir=%{lib}%/fftw3/lib"
> >        "--includedir=%{lib}%/fftw3/include"
> > -      "--datarootdir=%{lib}%/fftw3/share"]
> > +      "--datarootdir=%{lib}%/fftw3/share"] { os != "darwin" }
> > +  ["./configure"
> > +      "CFLAGS=-I/usr/local/include"
> > +      "LDFLAGS=-L/usr/local/lib"
>
> If I understand well, the default location to install FFTW3 on MacOS —
> including through a package manager — is /usr/local/?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ocaml/opam-repository/pull/5519/files#r51151104>.
>